result

```diff
@@ -0,0 +1 @@
+/nix/store/63pln8qafch9hhjfvp4lm6hha5db7vd3-upm-0.1.0-50e80cc
```

Comment:
This should probably not be added to the store. Maybe we need to add it to the `.gitignore`? :thinking:

Reply:
Oops!